src/map.rs

```diff
@@ -561,6 +561,13 @@ where
             }
         }
     }
+
+    fn contains_key(&self, k: &K) -> bool {
```

Comment:
This has to be a `pub fn` in order to be accessible from outside the crate, which I assume is the main motivation behind this PR.

Reply:
A documentation with some doc-comments (`///`) (similar to other `pub` functions) would be nice as well.

Reply:
Sorry about that, rookie mistake.  Will provide a new commit asap.